java/ql/src/Likely Bugs/Comparison/EqualsUsesInstanceOf.ql

```diff
@@ -17,8 +17,8 @@ predicate instanceofInEquals(EqualsMethod m, InstanceOfExpr e) {
   m.fromSource() and
   e.getEnclosingCallable() = m and
   e.getExpr().(VarAccess).getVariable() = m.getParameter() and
-  exists(Class instanceofType |
-    instanceofType = e.getTypeName().getType() and
+  exists(RefType instanceofType |
+    instanceofType = e.getCheckedType() and
```

Comment:
The change from `Class` to `RefType` is a behavior change, but it would probably result in more true positives.
Unless this was supposed to intentionally exclude interfaces?

Reply:
It does add a few results, which I haven't dug into, but the query is already `@precision low`, so it probably doesn't matter much.